WcaOnRails/lib/cutoff.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+class Cutoff
```

Comment:
Yeah, I am not wild about all the boilerplate these classes required to write them. Consolidating them into a "ValueType" class of some sort is something I'd be supportive of. I just want to make sure we're okay with using serializing here first.